docs/source/Notify/N001.rst

```diff
@@ -0,0 +1,212 @@
+.. include:: _notify_substitutions_n00x.repl
+
+.. _N001_page:
+
+Notifications Email (SMTP)
+===========================
+
+|N001_shortinfo|
+
+Type: |N001_type|
+
+Name: |N001_name|
+
+Status: |N001_status|
+
+GitHub: |N001_github|_
+
+Maintainer: |N001_maintainer|
+
+Used libraries: |N001_usedlibraries|
+
+Introduction
+------------
+
+*Notifications Email (SMTP)* is used to send an email message using ESPEasy rules.
+
+|
+
+Basic Rules Examples
+---------------------
+
+The general syntax for sending an email is:
+
+.. code-block:: none
+
+   notify <Nr>
+
+Where <Nr> is the position number (1,2,3) assigned to the notification declaration.
+
+For example, to send your pre-defined email message after WiFi has connected, use a rule like this:
+
+.. code-block:: none
+
+   on WiFi#Connected do
+      notify 1
+   endon
+
+To send an email every day at noon, create a rule like this:
+
+.. code-block:: none
+
+   on Clock#Time=All,12:00 do //will run once a day at noon
+      notify 1
+   endon
+
+.. hint::
+    The examples discussed here have setup the Notification Email in the very first (Nr 1) row. So the command syntax will be "*notify 1*"
+
+|
+
+Email Configuration
+--------------------
+
+ESPEasy's Notifications tab allows up to three declarations for email (and buzzer) alerts.
+
+.. image:: NotifyEmail1.png
+    :width: 600
```

Comment:
Did (finally) pull the changes local and generated and reviewed the resulting docs.

IMHO you can remove the `width` attribute on all images, as it effectively reduces the readability because of the downscaling, having them in full glory seems the best user-experience to me.

Reply:
I will remove the image width.

BTW, I prefer smaller images and click them when I need a bigger view. But different strokes for different folks.